nixos/modules/services/web-apps/invoiceplane.nix

```diff
@@ -0,0 +1,307 @@
+{ config, pkgs, lib, ... }:
+
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkMerge mkOption types;
+  inherit (lib) any attrValues flatten literalExpression nameValuePair;
+  inherit (lib) mapAttrs' mapAttrsToList concatStrings concatMapStringsSep;
```

Comment:
You don't think that using with lib or prefix all of these with lib is easer?

Reply:
Not sure, took it from Dokuwiki and Wordpress module https://github.com/NixOS/nixpkgs/blob/386234e2a61e1e8acf94dfa3a3d3ca19a6776efb/nixos/modules/services/web-apps/wordpress.nix#L4 , https://github.com/NixOS/nixpkgs/blob/386234e2a61e1e8acf94dfa3a3d3ca19a6776efb/nixos/modules/services/web-apps/dokuwiki.nix#L4
What would you prefer?

Reply:
`with lib;` is definitely better. Thanks for the hint!